build.gradle.kts

```diff
@@ -3,8 +3,8 @@ import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
 plugins {
     java
     id("com.github.ben-manes.versions") version "0.42.0"
-    kotlin("jvm") version "1.6.10"
-    kotlin("plugin.serialization") version "1.6.10"
+    kotlin("jvm") version "1.6.20"
+    kotlin("plugin.serialization") version "1.6.20"
```

Comment:
Should we pull the versions from the `Versions` object below?

Reply:
https://stackoverflow.com/a/60170218